content/chat/rooms/messages.textile

```diff
@@ -0,0 +1,80 @@
+---
+title: Messages
+meta_description: "Send and receive messages in chat rooms."
+product: chat
+languages:
+  - javascript
+---
+
+You can send and receive messages in a chat room with any number of participants. These users subscribe to messages by registering a listener, and send messages to all users that are subscribed to receive them.
+
+h2(#subscribe). Subscribe to messages
+
+Subscribe to receive messages in a room by registering a listener. Use the "@messages.subscribe()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/Messages.html#subscribe method in a room to receive all messages that are sent to it:
+
+```[javascript]
+const {unsubscribe} = room.messages.subscribe(message) => {
+  console.log(message);
+}
+```
+
+h3(#payload). Message payload
```

Comment:
(I feel like I wrote this comment before, sorry if it's a double)

Why _payload_? Makes me think of what's sent over realtime or similar. The **Chat Message** is a thing in itself, so there's no _payload_. There is a _payload_ over realtime message which is some data used to for the chat messages. This sounds weird, here's how it goes:
- over realtime, we send something like `data: {{text: "bla", metadata: {hello:"world"}}, extras: headers: {something: "else"}`
- realtime attaches more stuff to this, like the timestamp, timeserial, clientId
- so over realtime, a realtime message represents a chat message but nothing in that realtime message is exactly this _payload_
- when we receive those chat messages over realtime we convert the realtime payload with other info to create a Chat Message, which looks like what we have here under _payload_

Nothing here is wrong, just not sure about the word _payload_.

Same applies to reactions, and in the future other things we consider entities (as opposed to events).

Reply:
I see what you're saying, but I'm not sure I can think of a better word than 'payload' 🤔

Reply:
Just cut it and keep `Messasge`, or `The Message object`?

Reply:
(the irony is that we use `payload` in the readme as well)

Reply:
I've updated this to structure.